integreat_cms/cms/views/statistics/statistics_view.py

```diff
@@ -80,8 +90,9 @@ def get(
                 "form": form,
                 "pages": pages,
                 "region": region,
-                "language": region.default_language,
-                "languages": region.active_languages,
+                "language": language,
+                "languages": languages,
+                "accesses": access_legends,
```

Comment:
```suggestion
                "access_legends": access_legends,
```
Nit: I was initially confused by this naming in the template and think this could be more explicit :)